render_bouwkamp_code.py

```diff
@@ -46,51 +45,25 @@ class BouwkampCodeExtension(inkex.Effect):
     """
 
     def __init__(self):
-
         inkex.Effect.__init__(self)
-
-        self.OptionParser.add_option(
-            '--tab',
-            action='store',
-            type='string',
-            dest='current-tab'
-        )
-
-        self.OptionParser.add_option(
-            '--bouwkamp-code',
-            action='store',
-            type='string',
-            dest='bouwkamp_code',
-            default='21, 112, 112, [50, 35, 27], [8, 19], [15, 17, ' +
-            '11], [6, 24], [29, 25, 9, 2], [7, 18], [16], [42], [4, 37], [33]',
-            help='The Bouwkamp code.'
+        self.arg_parser.add_argument('--tab')
```

Comment:
Please restore the original indentation and line wrapping on the `add_argument`-calls.